README.md

```diff
@@ -850,6 +850,21 @@ deposit = starkbank.deposit.get("5155165527080960")
 print(deposit)
 ```
 
+## Update a deposit
+
+Update a deposit by passing its id to be partially or fully reversed.
```

Comment:
You can update an deposit's amount by its id. Only the amount can be decreased, which will result in a payment reversal. To fully reverse the deposit, pass amount = 0.